### ISSUE TITLE
Minimum Cuda capability is 3.5? But, 3.0 stated on site

### ISSUE THREAD

**Issue by mackiem** [tensorflow/tensorflow#17445]
I'm able to run the hello world examples, but the following warning (or error) is printed. So, I can't make use of my gpu? While this maybe a simple correction on the web page, is there anyway I can get a version that allows me to run with a Cuda 3.0 card?

OS: Ubuntu 16.04
GPU: K2000M

On the linux installation page, the minimum capability is written as 3.0. But, when I try to run hello world on a cuda 3.0 card, the following is printed:

name: Quadro K2000M major: 3 minor: 0 memoryClockRate(GHz): 0.745
pciBusID: 0000:01:00.0
totalMemory: 1.95GiB freeMemory: 977.81MiB
2018-03-05 13:43:54.533246: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1283] Ignoring visible gpu device (device: 0, name: Quadro K2000M, pci bus id: 0000:01:00.0, compute capability: 3.0) with Cuda compute capability 3.0. The minimum required Cuda capability is 3.5.

**Comment by yongtang** [comment]
I think minimal capability is 3.5 for binary install. It might still be possible to support 3.0 when building from source.

Create a PR #17448 for the doc fix.

**Comment by mackiem** [comment]
Ok, is this a regression? As per this thread this should be available?https://github.com/tensorflow/tensorflow/issues/25

**Comment by rohan100jain** [comment]
I think in the thread its mentioned that you might have to change some lines in common_runtime/gpu/gpu_device.cc for this to work. By default minimum is 3.5

**Comment by mackiem** [comment]
It can be compiled from source with CUDA capability 3.0. It is working for me this way.

**Comment by JoshuaC3** [comment]
@mackiem did you have to make the changes suggested by @rohan100jain before building from source?

**Comment by mackiem** [comment]
No. I believe basel configured it for me without having to touch the source.

**Comment by pabx06** [comment]
Hello i have a **CUDA Compute Capability of 2.1**  is there a chance i am able to change the source code to support my GPU lower capability? Or the tensorflow's algo need a specific capability ?

Cause CPU take way too long:  been training the model for more than 48h so far on **CPU**  
Likely i will be dying from old age before i can evaluate the inception model for my needs

- 19%-30% validation accuracy 
- 1.5 m training step
- model : inceptions-v3 transfert learning & retraining of classification layer
- dataset 6GB
- 130 classes, ... 
- GPU:Intel(R) Xeon(R) CPU E3-1220 V2 @ 3.10GHz

**Comment by mackiem** [comment]
Quite sure, 2.1 is too old for a lot of functionality and acceleration. Sorry.
